Batch8CucumberSeleniumAPI/src/test/resources/features/ui/Search.feature

```diff
@@ -23,4 +23,9 @@ Feature: Search functionality
 		When I search for product as "Dell"
 		And I click on any product in the Search Result
 		Then I am able to see product description and detail in new tab
-	  
\ No newline at end of file
+		
+		@TC0017
+		Scenario: A User entered any keyword in search then in Search dropdown it should display appropriate options which is entered in the Search.
+	  Given I have browser opened and url is navigated
```

Comment:
Plase keepa  proper indentation while writing the steps

Reply:
ok